jupyterhub/notebook-images/overlays/odh-ray-cluster/ray-custom-resources.yaml

```diff
@@ -0,0 +1,4320 @@
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
So with this we're effectively going to continue requiring cluster admin for the ODH operator if you want to install Ray.

Would it instead be better to have these CRDs installed by the user before deployment of Ray artifacts?

Reply:
@LaVLaS I'm thinking about our previous discussion where we said ODH wants to get out of the business of installing operators, instead focusing on deploying _custom resources_ for already installed operators

Reply:
ODH will no longer install operators that are available in OperatorHub. After doing a quick check, I don't see any OperatorHub packageManifest so this manual operator install would be the only option for now.

Reply:
yes, this is not available as the operator on operatorhub.